app/views/layouts/blacklight/base.html.erb

```diff
@@ -18,15 +18,15 @@
   <body class="<%= render_body_class %>">
     <%= render partial: 'shared/header_navbar' %>
 
-    <div id="main-container" class="<%= container_classes %>">
+    <div id="main-containerx" class="<%= container_classes %> mt-4">
```

Comment:
@cdmo `main-containerx` is x typo or intentional?

Reply:
typo, oops :(

Thanks for reading carefully!!!

Reply:
```suggestion
    <div id="main-container" class="<%= container_classes %> mt-4">
```